src/System.IO.FileSystem/pkg/System.IO.FileSystem.pkgproj

```diff
@@ -2,12 +2,18 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.2\System.IO.FileSystem.depproj">
```

Comment:
Yes, and by convention just use 4.0 since we won't have more than one 4.0.x version.